lib/ui/survey_question/survey_questions_screen.dart

```diff
@@ -107,11 +115,19 @@ class _SurveyQuestionsState extends ConsumerState<SurveyQuestionsScreen> {
   }
 
   Widget _buildCoverImageUrl(String largeCoverImageUrl) {
-    return Image.network(
-      largeCoverImageUrl,
-      fit: BoxFit.cover,
+    return Container(
       width: double.infinity,
       height: double.infinity,
+      decoration: BoxDecoration(
+        image: DecorationImage(
+          image: Image.network(largeCoverImageUrl).image,
+          fit: BoxFit.cover,
+        ),
+      ),
+      child: BackdropFilter(
+        filter: ImageFilter.blur(sigmaX: 10.0, sigmaY: 10.0),
+        child: const SizedBox(),
```

Comment:
Can we get rid of this? 🤔

Reply:
@hoangnguyen92dn, Without defining `child: const SizedBox()` We can't see the `BackdropFilter` So, let's keep it 🙏  

- With defining SizedBox:

![Simulator Screen Shot - iPhone 13 Pro Max - 2023-05-31 at 18 19 54](https://github.com/Wadeewee/survey-flutter-ic-pooh/assets/28002315/eb08cbc9-a885-4390-9428-e3a4a81b341c)

- Without defining SizedBox:

![Simulator Screen Shot - iPhone 13 Pro Max - 2023-05-31 at 18 20 30](https://github.com/Wadeewee/survey-flutter-ic-pooh/assets/28002315/785f3088-6357-41ec-8f47-7bdc6fab6b66)